source/extensions/tracers/datadog/tracer.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include <optional>
+#include <string>
+
+#include "envoy/thread_local/thread_local.h"
+#include "envoy/tracing/trace_driver.h"
+
+#include "source/common/common/logger.h"
+#include "source/extensions/tracers/datadog/tracer_stats.h"
+
+#include "datadog/tracer.h"
+#include "datadog/tracer_config.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace Datadog {
+
+/**
+ * Entry point for Datadog tracing. This class accepts configuration and runtime
+ * dependencies, and installs a thread-local instance of itself from which
+ * traces can be created.
+ */
+class Tracer : public Tracing::Driver, private Logger::Loggable<Logger::Id::tracing> {
+public:
+  /**
+   * Create a \c Tracer that produces traces according to the specified \p config
```

Comment:
what do these back-slash escaped sequences do?

Do they render well in some IDE? I have not noticed these used in Envoy.

Reply:
![Screenshot from 2023-03-27 16-56-05](https://user-images.githubusercontent.com/5205367/228064683-cef1cae1-0d9d-4c81-a93f-35eac6521f8d.png)

This is doxygen-style fixed-width formatting. I think that `\c` means "class" and `\p` means "parameter," though the doxygen docs point out that they can be used interchangeably.

My editor changes the color, and presumably rendered doxygen puts the following word in `<code></code>` tags.

Envoy code I've looked at doesn't use these operators. I started using them as a substitute for backticks when in an earlier review you advised me to comment doxygen-style.

I could remove them, but I like them. Up to you.